tools/bridge/end2end-tests/tests/test_transfer.py

```diff
@@ -85,6 +85,14 @@ def assert_within_timeout(check_function, timeout, poll_period=0.5):
                 break
 
 
+def assert_false_during_timeout(check_function, timeout, poll_period=0.5):
```

Comment:
What is the use of that?

Reply:
To assert something remains false during a period. Similarly to how you want to see an event occur within a time limit, you might want to check that the opposite event did not happen within the same time limit.

Reply:
So you mean
```
sleep(timeout)
assert
```
Why do you write it so complicated backwards?